etc/Makefile

```diff
@@ -0,0 +1,27 @@
+# Command aliases
+CONDA_EXE ?= conda
+CONDA_LOCK := $(CONDA_EXE)-lock
+
+name = $(word 1, $(subst ., ,$1))
+
+ENVIRONMENT_YMLS = $(wildcard *.environment.yml)
+ENVIRONMENTS := $(foreach e,$(wildcard *.environment.yml),$(call name, $e))
+LOCKFILES := $(foreach e, $(ENVIRONMENTS), $e.conda-lock.yml)
+# Explicit lockfiles are used in github actions with conda-incubator/setupminiconda
+EXPLICIT_LOCKFILES := $(foreach e, $(ENVIRONMENTS), $e.linux-64.lock)
+
+help:  ## Display help on all Makefile targets
+	@@grep -h '^[a-zA-Z]' $(MAKEFILE_LIST) | awk -F ':.*?## ' 'NF==2 {printf "   %-20s%s\n", $$1, $$2}' | sort
+
+.PHONY: $(ENVIRONMENTS)
+$(ENVIRONMENTS):  ## Lock this specific environment.yml file
```

Comment:
Not for now. But I think this might actually be a perfect use case NOT to use PHONY :)

Reply:
see #6